pkg/maps/lxcmap/lxcmap.go

```diff
@@ -192,8 +192,10 @@ func AddHostEntry(ip net.IP) error {
 // Returns boolean indicating if a new entry was added and an error.
 func SyncHostEntry(ip net.IP) (bool, error) {
 	key := NewEndpointKey(ip)
-	value, err := LXCMap().Lookup(key)
-	if err != nil || value.(*EndpointInfo).Flags&EndpointFlagHost == 0 {
+	v, err := LXCMap().Lookup(key)
+	value, ok := v.(*EndpointInfo)
```

Comment:
Should we return a specific error in case `ok == false`? Right now we'll end up at line 204 with `err == nil`.

Reply:
I've simplified this to be a bare assertion, because this cannot be false.